rebar.config.script

```diff
@@ -1,4 +1,11 @@
 %% -*- erlang-mode -*-
+case os:getenv("ERLANG_ROCKSDB_OPTS") of
```

Comment:
Would you clarify why we need this ? Isn't the upstream building without that option ? I would guess they have working defaults ?

Reply:
They don't, actually. The `mnesia_rocksdb` backend crashes on opening a table without some sort of compression included in the build.